samples/graph-meeting-notification/csharp/MeetingNotification/Controllers/NotificationController.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace MeetingNotification.Controllers
+{
+    using MeetingNotification.Helper;
+    using MeetingNotification.Helpers;
+    using MeetingNotification.Model;
+    using MeetingNotification.Model.Configuration;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Builder.Integration.AspNet.Core;
+    using Microsoft.Bot.Schema;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+    using Newtonsoft.Json;
+    using System;
+    using System.Collections.Concurrent;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Web;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class NotificationsController : Controller
+    {
+        /// <summary>
+        /// Stores the Bot configuration values.
+        /// </summary>
+        private readonly IOptions<BotConfiguration> botSettings;
+
+        private readonly ILogger _logger;
+        private readonly IBotFrameworkHttpAdapter _adapter;
+        private readonly ConcurrentDictionary<string, ConversationReference> _conversationReferences;
+
+        public NotificationsController(IOptions<BotConfiguration> botSettings,
+            ILogger<NotificationsController> logger,
+            IBotFrameworkHttpAdapter adapter,
+            ConcurrentDictionary<string, ConversationReference> conversationReferences)
+        {
+            this.botSettings = botSettings;
+            _logger = logger;
+            _adapter = adapter;
+            _conversationReferences = conversationReferences;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> Get()
+        {
+            return Json(SubscriptionManager.Subscriptions.Select(s => new
+            {
+                Resource = s.Value.Resource,
+                ExpirationDateTime = TimeZoneInfo.ConvertTimeBySystemTimeZoneId(s.Value.ExpirationDateTime.Value.DateTime, "UTC", "India Standard Time").ToString("MM/dd/yyyy h:mm tt"),
+                Expired = s.Value.ExpirationDateTime < DateTime.UtcNow
+            }));
+        }
+
+        // Callback
+        [Route("/api/webhookLifecyle")]
```

Comment:
What is this call back ?
From where we are going to call it and what is the purpose of this method ?

Reply:
Deleted not used anywhere.